recipes/qxlsx/all/test_package/CMakeLists.txt

```diff
@@ -8,4 +8,4 @@ find_package(QXlsx REQUIRED CONFIG)
 
 add_executable(${PROJECT_NAME} test_package.cpp)
 target_link_libraries(${PROJECT_NAME} QXlsx::Core)
-set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 11)
+set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 17)
```

Comment:
@prince-chrismc How shall I handle the C++ version conditionnally here ?

Reply:
Since this is the test packge, it's prefect